state/application.go

```diff
@@ -184,17 +184,17 @@ func (a *Application) SetAgentVersion(v version.Binary) (err error) {
 	if err = checkVersionValidity(v); err != nil {
 		return errors.Trace(err)
 	}
-	tools := &tools.Tools{Version: v}
+	result := &tools.Tools{Version: v}
```

Comment:
This confuses me - I'd expect result to be the name of the return value of the function.